content/help/02_crossing.md

```diff
@@ -1,9 +1,18 @@
-{{HELP_CROSSING_INFO}}[ref\_{{POLICE}}](https://www.police.hu/index.php/hu/hirek-es-informaciok/hatarinfo###3/18/2022) [{{DOWNLOAD_INFO}}](https://tcn.hu/wp-content/uploads/2022/03/TCN_ukrajnai_menekult_tajekoztato_V3_text_A4.pdf)
+{{_INFO}}[ref\_{{POLICE}}](https://www.police.hu/index.php/hu/hirek-es-informaciok/hatarinfo###3/18/2022) [{{DOWNLOAD_INFO}}](https://tcn.hu/wp-content/uploads/2022/03/TCN_ukrajnai_menekult_tajekoztato_V3_text_A4.pdf)
```

Comment:
labels should be `CROSSING_INFO` etc. instead of `_INFO` etc., or the translater needs to be updated. with the current implementation, the translation file contains the text as `sheet: CROSSING, label: _INFO` but the translater resolves it as `CROSSING_INFO` when building the final labels.

Reply:
upsz. my bad. 
I suggest to change the google sheet name to "HELP_CROSSING" I think that makes more understandable the crossing.md file and the google sheet as well.

Reply:
good idea, however that will require some changes from the translater as well. I'll add it to the issues